google/cloud/storage/internal/grpc_object_read_source.cc

```diff
@@ -94,7 +97,7 @@ StatusOr<ReadSourceResult> GrpcObjectReadSource::Read(char* buf,
     }
   };
 
-  spill_ = update_buf(std::move(spill_));
+  spill_view_ = update_buf(std::move(spill_view_));
```

Comment:
Nit: `absl::string_view` will never (I claim) have special move behavior, so it probably isn't worth indicating that it might.